### PR TITLE
Feature/add scrutinizer

### PR DESCRIPTION
**Pull request by GeeH**

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        enabled: false
+    php_code_sniffer:
+        enabled: true
+        config:
+            standard: PSR2
+        filter:
+            paths: ["module/*"]
+    php_cpd:
+        enabled: true
+        excluded_dirs: ["docs", "examples", "tests", "vendor", "data"]
```
Comment (localheinz) [inline .scrutinizer.yml]: @GeeH 

We don't have `docs` or `examples` directories, do we?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# ZF2 Modules Site [![Build Status](https://travis-ci.org/zendframework/modules.zendframework.com.svg?branch=master)](https://travis-ci.org/zendframework/modules.zendframework.com) [![Dependency Status](https://www.versioneye.com/user/projects/54885d5a746eb514b0000279/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54885d5a746eb514b0000279)
+# ZF2 Modules Site [![Build Status](https://travis-ci.org/zendframework/modules.zendframework.com.svg?branch=master)](https://travis-ci.org/zendframework/modules.zendframework.com) [![Dependency Status](https://www.versioneye.com/user/projects/54885d5a746eb514b0000279/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54885d5a746eb514b0000279) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/zendframework/modules.zendframework.com/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/zendframework/modules.zendframework.com/?branch=develop) [![Build Status](https://scrutinizer-ci.com/g/zendframework/modules.zendframework.com/badges/build.png?b=develop)](https://scrutinizer-ci.com/g/zendframework/modules.zendframework.com/build-status/develop)
```
Comment (localheinz) [inline README.md]: @GeeH 

The links refer to the `develop` branch, should be `master`, right?

File: .scrutinizer.yml (hunk 1 above)
Comment (GeeH) [inline .scrutinizer.yml]: no, it's a standard config I use across projects and does no harm

File: README.md (hunk 2 above)
Comment (GeeH) [inline README.md]: Yes, I thought I'd changed that using the github editor, let me try again

**Comment by localheinz** [comment]
@GeeH 

Something's still wrong with the Scrutinizer setup, can you have a look?

* https://scrutinizer-ci.com/g/zendframework/modules.zendframework.com/inspections/39f884ff-eb8b-48de-a3c2-8c8093999a61

File: .scrutinizer.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    php_cs_fixer:
+        enabled: true
+        config:
+            level: all
+        filter:
+            paths: ["module/*"]
```
Comment (localheinz) [inline .scrutinizer.yml]: @GeeH 

I guess we should not run `php-cs-fixer` on Travis, then?

Comment (Ocramius) [inline .scrutinizer.yml]: Scrutinizer is not going to fail - we want a failure on CS issues

File: .scrutinizer.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+before_commands:
+    - "composer install --no-dev --prefer-source"
+
+tools:
+    external_code_coverage:
```
Comment (Ocramius) [inline .scrutinizer.yml]: Note: needs ocular to submit coverage. See https://github.com/Ocramius/ProxyManager/blob/master/.travis.coverage.sh

File: .scrutinizer.yml (hunk 3 above)
Comment (Maks3w) [inline .scrutinizer.yml]: @Ocramius https://scrutinizer-ci.com/docs/configuration/build_status

**Comment by Maks3w** [comment]
This proposal should be raised to crteam ML
